eventdata/challenges/daily-log-index-fixed-throughput-and-query.json

```diff
@@ -0,0 +1,137 @@
+{% set p_bulk_indexing_clients = (bulk_indexing_clients | default(8)) %}
+{% set p_bulk_size = (bulk_size | default(1000)) %}
+{% set p_bulk_indexing_reqs_per_sec = (bulk_indexing_reqs_per_sec | default(20)) %}
+{% set p_bulk_indexing_throughput = (p_bulk_indexing_reqs_per_sec * p_bulk_size) %}
+{% set p_search_clients = (search_clients | default(1)) %}
+{% set p_starting_point = (starting_point | default("2018-05-25 00:00:00")) %}
+{% set p_number_of_days = (number_of_days | default(6)) %}
+{% set p_daily_logging_volume = (daily_logging_volume | default("100GB")) %}
+
+{#
+  This challenge assumes that `index-logs-fixed-daily-volume` has been executed before.
+#}
+
+{
+  "name": "index-fixed-load-and-query",
+  "description": "Indexes {{ p_number_of_days }} days of logs (each {{ p_daily_logging_volume }} size) running at {{ p_bulk_indexing_throughput }} docs/s while running queries concurrently",
+  "meta": {
+    "client_count": {{ p_bulk_indexing_clients }},
+    "benchmark_type": "logs-fixed-index-daily-throughput-and-query"
+  },
+  "schedule": [
+    {
+      "name": "check-cluster-health",
+      "operation": {
```

Comment:
we should add a `"retry-until-success": true` here (similar to https://github.com/elastic/rally-eventdata-track/pull/97).